### ISSUE TITLE
BUG: Error serializing object

### ISSUE THREAD

**Issue by dantleech** [phpbench/phpbench#843]
```
 PhpBench\Executor\Exception\ExecutionError PHP Fatal error:  Uncaught Error: Call to undefined method Phpactor
\TextDocument\StandardTextDocument::__set_state() in /tmp/daniel/PhpBench1zAxEd:68                                    Stack trace:                                                                                                  
    #0 {main}                                                                                                           thrown in /tmp/daniel/PhpBench1zAxEd on line 68 
```

**Comment by dantleech** [comment]
Requires refactoring, to fix in 1.1

**Comment by dantleech** [comment]
Fixed with #845
